src/components/question/question.html

```diff
@@ -0,0 +1,153 @@
+    <link
+      rel="mask-icon"
+      href="../../../assets/safari-pinned-tab.svg"
+      color="#5bbad5"
+    />
+    <meta name="msapplication-TileColor" content="#da532c" />
+    <meta name="theme-color" content="#ffffff" />
+  </head>
+  <body>
+    <div class="pagewrapper">
+      <!-- HEADER NAVIGATION -->
+      <nav class="nav-container">
+        <div class="brand">
+          <a href="../../../index.html"
+            ><img src="/src/images/logo.webp" alt="muffin-logo" class="logo"
+          /></a>
+          <a href="../../../index.html" class="brand-name">Bakin Mania</a>
+        </div>
+        <ul>
+          <li class="nav-icon">
+            <a href="#" rel="noreferrer" aria-label="search">
+              <i class="fa fa-solid fa-search fa-lg nav-icon"></i>
+            </a>
+          </li>
+          <li class="nav-icon">
+            <a href="#" rel="noreferrer" aria-label="account">
+              <i class="fas fa-trophy fa-lg nav-icon"></i>
+            </a>
+          </li>
+          <li class="nav-icon">
+            <a
+              href="../authentication/login.html"
+              rel="noreferrer"
+              aria-label="account"
+            >
+              <i class="fa fa-solid fa-user fa-lg nav-icon"></i>
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+      <!-- MIDDLE SECTION STARTS-->
+      <!-- CARDS SECTION -->
+      <main class="container-main flex-row-center">
+        <div class="container-question flex-column-center">
+          <div class="question-header flex-row-center">
+            <div class="question-number">Question 1</div>
+            <div class="timer">
+              <i class="fas fa-stopwatch keyword"></i>
+              <span>00:59</span>
+            </div>
+          </div>
+          <div class="question-and-options flex-column-center">
+            <div class="question">
+              Which common cake ingredient can be replaced with a mashed banana?
+            </div>
+            <div class="options flex-column-center">
+              <label class="option-label">
+                <input type="radio" class="option" name="option" />Eggs
+              </label>
+              <label class="option-label">
+                <input type="radio" class="option" name="option" />Butter
+              </label>
+              <label class="option-label">
+                <input type="radio" class="option" name="option" />Milk
+              </label>
+              <label class="option-label">
+                <input type="radio" class="option" name="option" />Flour
+              </label>
+            </div>
+            <button class="btn btn-primary">Next >></button>
+          </div>
+        </div>
+      </main>
+      <!-- MIDDLE SECTION ENDS-->
+
+      <!-- FOOTER -->
+      <footer class="footer-container flex-column-center">
+        <p class="footer-header">
+          Made with <span>&lt;/&gt;</span> by Roma Bulani
```

Comment:
we can look for a similar way as template literals to integrate symbols

Reply:
I looked for multiple blogs and resources, they have done using template literals using jsx, will change these when we will add jsx